lib/normalizer.c

```diff
@@ -1696,6 +1699,10 @@ grn_nfkc_normalize_remove_character_p(grn_ctx *ctx,
                                       const unsigned char *current,
                                       size_t current_length)
 {
+  if (data->options->char_type_func(current) == GRN_CHAR_SYMBOL) {
```

Comment:
Are unprintable ascii caught at this `if`?
If so, it may change original behavior about unprintable ascii.
e.g. They are not removed when `data->remove_symbol_p` is `false`.

Reply:
Ah,I noticed that the type of unprintable ascii is `GRN_CHAR_BRANK`.

https://github.com/groonga/groonga/blob/master/lib/normalizer.c#L1898

Reply:
Ah, I noticed that unprintable ascii are regard as `GRN_CHAR_OTHERS` in the NFKC normalizers and not be caught at this `if`.
It seems good.